script/generateUtilsDoc.sh

```diff
@@ -28,8 +29,11 @@ touch docs/utils.html
 pod2html --infile=lib/utils.pm --outfile=docs/utils.html
 
 #checkout old docs and compare to new ones, then toggle flag accordingly
-git diff gh-pages -- docs/utils.html
+git checkout gh-pages utils.html
+diff -u utils.html docs/utils.html
```

Comment:
iirc there will be a line in the html which is always changing...

Reply:
That line is now taken care of.